extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -120,20 +120,3 @@ export function convertSlashesForSqlProj(filePath: string): string {
 	const parts = filePath.split('/');
 	return parts.join('\\');
 }
-
-/**
- * Read SQLCMD variables from xmlDoc and return them
- * @param xmlDoc xml doc to read SQLCMD variables from. Format must be the same that sqlproj and publish profiles use
- */
-export function readSqlCmdVariables(xmlDoc: any): Record<string, string> {
```

Comment:
doesn't project.ts use this to read the variables in the .sqlproj?

Reply:
no actually nothing uses it. Project uses the one from profile - I can move it here and remove the profile one

Reply:
moved to util removed from publishprofile

Reply:
ah ok oops I must've forgotten to move it when moving the publish profile stuff to its own file. Thanks for cleaning up!